### PR TITLE
48 stack navigator

### LINKED ISSUE #48: NAVIGATION: Stack Navigator

### PR DESCRIPTION
**Pull request by JeffKandel**
closes #48

### CODE REVIEW

File: app/redux/reducers/tasks.js [hunk 1]
```diff
@@ -46,7 +46,7 @@ export const addSelectedTask = selectedTask => {
 };
```
Comment (dffattal) [inline app/redux/reducers/tasks.js:50]: ipAddress and port constants are not being imported into this file.

**Review by dffattal — APPROVED** [review]
Good work on refactoring the navigation, looks good.

**Review by miggitymack — APPROVED** [review]
We might need implement back buttons because of the lack of tabs

**Comment by JeffKandel** [comment]
@miggitymack there is a card for back button functionality.

**Pull request merged**